codespell_lib/data/dictionary_code.txt

```diff
@@ -29,6 +29,7 @@ falsy->falsely, false,
 gadjet->gadget
 gadjets->gadgets
 gae->game, Gael, gale,
+grap->grep
```

Comment:
This can go in the main dictionaries (unless our CI complains that grep isn't a word.

The code one is to protect words you'd find in code from being corrected when they shouldn't. e.g. grep->grape would go in here, as you wouldn't want to run it over code, but if you were just writing prose it should be safe.

Reply:
Ok, I moved this rule.